minorminer/utils/raster_embedding.py

```diff
@@ -199,21 +199,36 @@ def raster_embedding_search(S, T, timeout=10, raster_breadth=None,
             return []
     # A possible feature enhancement might allow for raster_breadth to be
     # replaced by raster shape.
-    if T.graph.get('family') == 'chimera':
+    family = T.graph.get("family")
+    if family == 'chimera':
         sublattice_mappings = dnx.chimera_sublattice_mappings
         t = T.graph['tile']
-        tile = dnx.chimera_graph(m=raster_breadth, n=raster_breadth, t=t)
-    elif T.graph.get('family') == 'pegasus':
+        if tile is None:
+            tile = dnx.chimera_graph(m=raster_breadth, n=raster_breadth, t=t)
+    elif family == 'pegasus':
         sublattice_mappings = dnx.pegasus_sublattice_mappings
-        tile = dnx.pegasus_graph(m=raster_breadth)
-    elif T.graph.get('family') == 'zephyr':
+        if tile is None:
+            tile = dnx.pegasus_graph(m=raster_breadth)
+    elif family == 'zephyr':
         sublattice_mappings = dnx.zephyr_sublattice_mappings
         t = T.graph['tile']
-        tile = dnx.zephyr_graph(m=raster_breadth, t=t)
+        if tile is None:
+            tile = dnx.zephyr_graph(m=raster_breadth, t=t)
     else:
-        raise ValueError("source graphs must a graph constructed by "
+        raise ValueError("source graphs must a Zephyr graph constructed by "
                          "dwave_networkx as chimera, pegasus or zephyr type")
 
+    # TO DO: Add a function that finds the minimal feasible raster_breadth for
```

Comment:
You need to rebase to code_dev. These lines need to be deleted. In the most recent version of code_dev these lines are deleted and shouldn't be put back in.

Reply:
Or just delete 221 - 228. These lines are made redundant by 196-198